### ISSUE TITLE
Proposal: create "ACT Heuristics" as an "Accessibility Requirements Document"

### ISSUE THREAD

**Issue by nitedog** [act-rules/act-rules.github.io#428]
Currently the proposed rule "CSS Orientation Lock" (PR #408) is a good example of using the Assumption section to essentially redefine the accessibility requirement, which I think is a misuse.

Currently this rule only checks the use of CSS for orientation lock. It will claim that any such occurrence fails to meet WCAG 2.1 SC 1.3.4, which is technically inaccurate. A rule for SC 1.3.4 must consider other aspects, such as potentially widgets on the page that change the orientation lock. It must also consider the exception "unless a specific display orientation is essential". However, that would require a composite rule that is rather complex and manual. A counter-argument is the using CSS for orientation lock is most commonly a failure, and catching these automatically is of high value in testing.

A suggested compromise would be for this group (currently Auto-WCAG CG, soon ACT Rules CG) to create and maintain an "ACT Heuristics" document with a set of "requirements" that frequently indicate WCAG conformance failures but not always. Examples of such requirements could include:

- Use of CSS for orientation lock
- Wrong use of WAI-ARIA properties
- ...

Essentially we would move interpretations that we currently have within the Assumption section of the rule into a stand-alone document. This document would meet the definition of [Accessibility Requirements Document](https://pr-preview.s3.amazonaws.com/w3c/wcag-act/pull/328.html#accessibility-requirements-document) in the upcoming draft of the ACT Rules Format.

Rules, such as CSS Orientation Lock, would refer to that "requirement" in the ACT Heuristics document rather than directly to WCAG. This rule could also become part of a more comprehensive composite rule, which fully covers SC 1.3.4 and can therefore directly refer to WCAG.

I think this approach would keep the rules "clean" from any (mis)interpretations of WCAG, while at the same time documenting valuable heuristics/indicators that can be used for testing.

**Comment by WilcoFiers** [comment]
This issue has been discussed. Assumptions are the primary way to address potential false positives. If you'd like to revisit this issue feel free to reopen this Shadi.
